.github/workflows/CI.yml

```diff
@@ -69,12 +56,14 @@ jobs:
           cd ..
       - name: Install dependencies
         run: |
-          apm install
-          apm install linter-eslint
-          # ./node_modules/.bin/atom-package-deps .
+          pulsar --package install
```

Comment:
I initially tried to use `ppm` here, but it failed on macos.

I found that its disabled here, hot sure why:
https://github.com/pulsar-edit/action-pulsar-dependency/blob/main/action.yml#L27

Reply:
@confused-Techie, do you remember why? ^^

Reply:
This should only be commented out, [in case someone wanted to symlink `ppm`](https://github.com/pulsar-edit/action-pulsar-dependency/commit/7d1241c42abbc265a05f40c2dade61c655043ff9), but the idea is we move fully over to using `pulsar -p`.

On a side note, @savetheclocktower is 100% right, you must still use `apm` on Windows. Plus the installer doesn't add anything to the path. You could run the tests from the path of the package after install, like I do [here](https://github.com/confused-Techie/language-robots-txt/blob/main/.github/workflows/test.yml), when running on windows

Reply:
mmm, I ended up using a patched version `atom-package-deps` to install the packages on windows 🤔

I probably need to work out a better solution 😬